t/t0018-content-files.t

```diff
@@ -73,6 +73,10 @@ kvs_checkpoint_get() {
 # Tests of the module by itself (no content cache)
 ##
 
+test_expect_success 'content-files module load fails with unknown option' '
+	test_must_fail flux module load content-files notoption
+'
+
```

Comment:
nit, should this test be a different commit? not really related to ` content-files: add truncate module option `